content/guides/requesting-pre-auth.md

```diff
@@ -0,0 +1,115 @@
+---
+title: Requesting Pre Auth
+---
+
+Centrapay’s Pre Auth extension allows a patron to authorise payment up to a limit when the actual payment amount is not yet known.
+
+## Restrictions
+
+Pre Auth payments are not supported in all cases.
+
+1. Not all asset types support Pre Auth - Payment options for asset types that do not support Pre Auth will be excluded when a payment request is created with the `preAuth` flag.
+2. Pre Auth is incompatible with Multi-Asset payments - Only one asset type can be authorised for a Pre Auth.
+
+## Pre Auth Flow
+
+Pre Auth payments go through an orthogonal payment flow compared to Centrapay’s standard payment flow when [Requesting Payment](https://www.notion.so/Requesting-Payment-ad4c917a690a4bc3a4de5fc04a7396c2).
+
+```mermaid
+flowchart LR
```

Comment:
This does not automatically have the chart centred, as pictured in PR description. 
I cannot see a way to configure this when initialising mermaid so could add it as a tilde task.